build.gradle

```diff
@@ -18,5 +18,6 @@ def versionLabel(gitInfo) {
 
 allprojects {
     group = 'gov.ca.dwr'
-    version = versionLabel(versionDetails())
+    version = '1.0.0.0' //hard coding version to match wrims gui expectations.
+//    version = versionLabel(versionDetails())
```

Comment:
This will likely break publishing stuff in the near future, we'll probably need to find a way to allow changing versions. Maven repositories don't support multiple revisions of a single artifact / version

Reply:
We will resolve the versioning after the installer is successfully deploying through github. This issue is being tracked under: https://github.com/CentralValleyModeling/wrims/issues/149